### DISCUSSION TITLE
OSX should changed to macOS

### DISCUSSION THREAD

**Discussion by 18o** [aristocratos/btop#174] [General]
current the latest version of operating system on mac is 12 which is called `macOS` not `OSX`, since version 11 `OSX` has been discarded

**Comment by aristocratos** [comment]
Since more versions of OSX 10 are supported than the 2 that are 11 and 12, OSX would still be more apt. And if you look at bit more you would notice that the binaries for 11 and 12 are named `macos` including the binaries from the "Continuous build Macos".

If apple have decided to also retroactively rename the older versions to macos, I can't say I really care. Anyone confused to what OSX would mean should be able to by deduction decipher it from the text in the readme where both OSX and Macos are mentioned.
